common/changes/@microsoft/rush/will-subspace-patch-4_2024-02-27-22-23.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix issue where rush looks through all projects instead of subspace projects",
```

```suggestion
      "comment": "Fix an issue where, when the experimental subspaces feature was enabled, the lockfile validation would check irrelevant subspaces",
```

@william2958 FYI - these changelogs should clarify that they only apply to people who enabled the experiment